lightning/src/ln/channelmanager.rs

```diff
@@ -1150,6 +1150,10 @@ pub struct CounterpartyForwardingInfo {
 	/// such that the outgoing HTLC is forwardable to this counterparty. See `msgs::ChannelUpdate`'s
 	/// `cltv_expiry_delta` for more details.
 	pub cltv_expiry_delta: u16,
+	/// The maximum amount of msats that can be forwarded by a channel holder in an HTLC.
+	pub holder_htlc_maximum_msat: u64,
```

Comment:
Hmm, this struct is about settings our counterparty sets on HTLCs we forward outbound, I don't think `holder_htlc_maximum_msat` belongs here, but `counterparty_htlc_maximum_msat` probably does.